app/javascript/topNavigation/__tests__/utilities.test.js

```diff
@@ -142,10 +142,10 @@ describe('top navigation utilitities', () => {
     });
   });
 
-  describe('initializeTouchDevice', () => {
+  describe('initializeMemberMenu', () => {
     const initialMenuHTML = `
       <div class="crayons-header__menu" id="crayons-header__menu" data-testid="menu-dropdown">
-        <button type="button" class="crayons-header__menu__trigger" id="member-menu-button" aria-label="Navigation menu">
+        <button type="button" class="crayons-header__menu__trigger" id="member-menu-button" aria-expanded="false" aria-label="Navigation menu">
```

Comment:
The `aria-expanded` attribute needs to start off as `"false"` in the markup, like in the real app.

This should be converted to an E2E tests as mentioned, but I will do this at a later time as the tests are still relatively decent.